napari/_qt/_qapp_model/_menus.py

```diff
@@ -1,13 +1,22 @@
+from typing import Optional
+
 from app_model.backends.qt import QModelMenu
+from qtpy.QtCore import QObject
 
 
-def build_qmodel_menu(menu_id: str) -> QModelMenu:
+def build_qmodel_menu(
+    menu_id: str, title: str = None, parent: Optional[QObject] = None
+) -> QModelMenu:
     """Build a QModelMenu from the napari app model
 
     Parameters
     ----------
     menu_id : str
         ID of a menu registered with napari._app_model.get_app().menus
+    title : str
+        Title of the menu
+    parent : QObject
+        Parent of the menu
```

Comment:
Why this parameter is added? 
Or on the other hand. 
When a new contributor should set parent?

Reply:
see, for example, `self.view_menu = build_qmodel_menu` in qt_main_window

Reply:
But still, this requires `QWidget`, not `QObject`. 
```suggestion
    parent : QWidget
        Parent of the menu
```

Reply:
Oh I see, it was the type hint at question?